src/serialization/sb2.js

```diff
@@ -390,47 +390,20 @@ const parseMonitorObject = (object, runtime, targets, extensions) => {
 };
 
 /**
- * Parse a single "Scratch object" and create all its in-memory VM objects.
- * TODO: parse the "info" section, especially "savedExtensions"
+ * Parse the assets of a single "Scratch object" and load them.
  * @param {!object} object - From-JSON "Scratch object:" sprite, stage, watcher.
```

Comment:
This comment probably needs to change... this function shouldn't get called on watchers at the very least...

Reply:
Hmm. Good point.